staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -873,6 +873,9 @@ const (
 	// FieldManagerConflict is used to report when another client claims to manage this field,
 	// It should only be returned for a request using server-side apply.
 	CauseTypeFieldManagerConflict CauseType = "FieldManagerConflict"
+	// CauseTypeResourceVersionTooLarge is used to report that resource version is coming
+	// from the future and request cannot be served.
```

Comment:
"resource version is coming from the future" is confusing... consider rephrasing to something like

> CauseTypeResourceVersionTooLarge is used to report that that requested resource version is newer than the data observed by the API server, so the request cannot be served.